source/Camera/Basic/Capture/Capture.cpp

```diff
@@ -20,7 +20,7 @@ int main()
         const auto settings =
             Zivid::Settings{ Zivid::Settings::Acquisitions{ Zivid::Settings::Acquisition{
                                  Zivid::Settings::Acquisition::Aperture{ 5.66 },
-                                 Zivid::Settings::Acquisition::ExposureTime{ std::chrono::microseconds{ 8333 } } } },
+                                 Zivid::Settings::Acquisition::ExposureTime{ std::chrono::microseconds{ 6500 } } } },
```

Comment:
Could you mention in the commit message why each of the settings you use here are considered preferable?

Reply:
Changed to const auto